packages/dx-react-grid-bootstrap4/src/templates/column-chooser/container.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Container } from './container';
+
+describe('Container', () => {
+  it('should pass the className prop to the root element', () => {
+    const tree = shallow((
+      <Container
+        className="custom-class"
+      >
+        <div />
+        <div />
```

Comment:
Why do we need two `<div />` elements in tests?

Reply:
Because in propTypes write `children: PropTypes.arrayOf(PropTypes.node).isRequired`